UndertaleModLib/Compiler/Compiler.cs

```diff
@@ -54,6 +56,22 @@ public void OnSuccessfulFinish()
                 {
                     foreach (KeyValuePair<string, string> v in GlobalVars)
                         Data?.Variables?.EnsureDefined(v.Key, UndertaleInstruction.InstanceType.Global, false, Data.Strings, Data);
+                    if (Data != null) {
```

Comment:
Use `is not null` please. Not just here, but everywhere

Reply:
done